src/content/en/2020/css.md

```diff
@@ -1268,27 +1338,42 @@ There is so much discussion (or argument) about CSS-in-JS that one could assume
 
 ## Internationalization
 
+{# TODO Authors: Intro sentence #}
+
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - could you add something here?

Reply:
Hmm. OK here is an intro sentence.

"English, like many languages, is written in horizontal lines and the characters are laid out from left to right. But some languages (such as Arabic and Hebrew) are mostly written right to left. And then there are languages which may be written in vertical lines, from top to bottom. Not to mention quotations from other languages. So things can get quite complicated. Both HTML and CSS have ways to handle this."

Reply:
Added that in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1846/commits/d83e68afcf451c0735bddf8b527fb0b578be9435. Marking as resolved.